1-exercises/C-while-loop-with-array/exercise.js

```diff
@@ -17,7 +17,12 @@ const BIRTHDAYS = [
 ];
 
 function findFirstJulyBDay(birthdays) {
-    // TODO
+    let firstJulyBirthday= '';
+    for (let i=0; i<birthdays.length; i++){
+        if (birthdays[i] === 'July 11th')
```

Comment:
This checks for the 11th of July only, is there a way we could use includes method on arrays to check for any day in July?

Here's the docs for includes https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes